src/output-json-tls.c

```diff
@@ -478,7 +479,7 @@ static int JsonTlsLogger(ThreadVars *tv, void *thread_data, const Packet *p,
     }
     /* log extended */
     else if (tls_ctx->flags & LOG_TLS_EXTENDED) {
-        JsonTlsLogJSONExtended(js, ssl_state);
+        JsonTlsLogJSONExtended(js, ssl_state, false);
```

Comment:
if ja4 isn't logged in extended, how can it be logged outside of alerts?

Reply:
For 7.0.x, we decided that the ja4 hash would *only be logged on an alert (if the rule used JA4).

Am I misunderstanding your comment or our decision?

Reply:
I think we should still be able to log it, however not by default. So I think the patch gets the default behavior right, but we still need a way to allow the user to force log it with all relevant records.